spec/defines/keepalived_vrrp_track_process_spec.rb

```diff
@@ -0,0 +1,104 @@
+require 'spec_helper'
+
+describe 'keepalived::vrrp::track_process', type: :define do
+  let(:facts) { debian_facts }
```

Comment:
To not have to mock facts manualy, the module uses now [rspec-puppet-facts](https://github.com/mcanevet/rspec-puppet-facts). With this, the module is now tested on all supported OSes defined in `metadata.json` and not only Debian. 
The `README.md` of `rspec-puppet-facts` gives guidance on how to use it. And you can have a looks to tests in this module to get examples.